### ISSUE TITLE
[SF Migration] Document Filters component

### ISSUE THREAD

**Issue by sarjon** [PrestaShop/PrestaShop#10370]
As a reminder for @mickaelandrieu :wink:

**Comment by matks** [comment]
@sarjon I guess this task will be on me now :) would you please tell me what should be done ?

Is this about adding docs to explain how grid filters work ?

**Comment by sarjon** [comment]
not really, its more about filters without grid, the one you can inject into controller actions.

**Comment by matks** [comment]
@sarjon I'm sorry I cannot see the topic. Do you have an example ?

**Comment by sarjon** [comment]
its about filters and their definition, for example:

```php
class SomeController {
    public function indexAction(MyFilters $filters) {}
}
```
Here's definition example:
https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Search/Filters/LogsFilters.php

But they are pretty limited now, see old issue on prestafony https://github.com/PrestaShop/prestafony-project/issues/61

**Comment by mickaelandrieu** [comment]
Hello @sarjon,

as they must evolve in order to migrate Manufacturers page, can you ping me for the documentation once you will have contributed and improved this component?

Thanks
